evap/grades/migrations/0009_translate_description.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.1 on 2016-01-04 18:32
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grades', '0008_translate_description'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='gradedocument',
+            unique_together=set([('course', 'description_de'), ('course', 'description_en')]),
+        ),
+        migrations.RemoveField(
+            model_name='gradedocument',
+            name='description',
+        ),
+    ]
```

Comment:
@Paul-Geppert do you want to try this? what i'd do is start over with the base commit, rename description_de, make it a CharField, create CharField description_en, create migration. then add the data migration, and then add the second unique_together and create another migration. do a commit, and then somehow replace the migrations in your second commit with the new ones.

if you want to do something else, let me know, then i'd do this. or we look at it together during the next hacking hours.